app/server/models/User.js

```diff
@@ -114,7 +109,7 @@ var profile = {
   whyBitcamp: {
     type: String,
     min: 0,
-    max: 2500
```

Comment:
Why are we switching back to 250?

Reply:
This is only supposed to be a 250 character response, as outlined in the google doc. The other response is supposed to be a 100 word response, so it's set to be 1000 characters.